resolution-aware-images.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Resolution Aware Images
+permalink: /resolution-aware-images/
+---
+
+Flutter can load resolution-appropriate images for the current device
+pixel ratio.
+
+*Tip*: Remember to declare all of your resources inside `flutter.yaml`,
+as shown in the [Tutorial](/tutorial/).
+
+AssetImage understands how to map a logical requested asset onto one that most
+closely matches the current device pixel ratio. In order for this mapping to
+work, assets should be arranged according to a particular directory structure:
+
+    .../image.png
+    .../Mx/image.png
+    .../Nx/image.png
+    ...etc.
+
+
+Where M and N are numeric identifiers that correspond to the nominal resolution
+of the images contained within. The main asset is assumed to correspond to a
+resolution of 1.0. For example, consider the following asset layout for an
+image named `my_icon.png`:
+
+    .../my_icon.png
+    .../2.0x/my_icon.png
+    .../3.0x/my_icon.png
+
+On devices with a device pixel ratio of 1.8, the asset `.../2.0x/my_icon.png`
+would be chosen. For a device pixel ratio of 2.7, the asset
+`.../3.0x/my_icon.png` would be chosen.
+
+If the width and height of the rendered image are not specified, the nominal
+resolution is used to scale the asset so that it will occupy the same amount
+of screen space as the main asset would have, just with a higher resolution.
+That is, if `.../my_icon.png` is 72px by 72px, then `.../3.0x/my_icon.png`
+should be 216px by 216px; but they both will render into 72px by 72px
+(in logical pixels) if width and height are not specified.
+
+The way this works is through an object called AssetVendor established at the
+top of the build tree.  AssetVendor replaces the default asset bundle, so
+really anything using the default asset bundle will inherit resolution awareness
+when loading images.  If you work with some of the lower level classes, like
+ImageResource or ImageCache, you'll also notice some new parameters related
```

Comment:
For a general introduction I might even lead with AssetVendor, since AssetBundle is only one of several clients to the asset bundle loading system that inherits from AssetVendor's resolution logic.